packages/turbo/src/types.ts

```diff
@@ -27,10 +27,11 @@ export type MessageEvent = object;
 export interface AlertHandler {
   message: string;
 }
-export interface VisitProposalError {
-  statusCode: number;
+
+export interface ErrorEvent {
```

Comment:
I think we'd benefit greatly from surfacing the type of the error here as well to allow more granular control. On iOS the `TurboError` is an enum, so we should be able to be more specific about the type here.

Reply:
In theory we could always return the statusCode, which would then be `0` for `networkFailure` etc.